xOpera-rest-blueprint/modules/docker/playbooks/add_cert.yml

```diff
@@ -71,17 +63,5 @@
         - command: "cp /home/xopera/certs/image.docker.local.key /etc/docker/certs.d/{{ registry_ip }}"
         - command: "cp /home/xopera/certs/image.docker.local.crt /etc/docker/certs.d/{{ registry_ip }}/image.docker.local.cert"
 
-    - name: Configure the docker mtu for OpenStack and restart docker host
```

Comment:
I would leave this - since there docker usually sets MTU to 1500 which occasionally creates unsuspected problems in communication with openstack instances..